samples/bluetooth/bthome_sensor_template/README.rst

```diff
@@ -0,0 +1,23 @@
+.. _bluetooth-bthome-sensor-template-sample:
+
+Bluetooth: BTHome sensor template
+#################################
+
+Template for a `BTHome <https://bthome.io/>`_ sensor.
+
+Requirements
+************
+
+* A board with BLE support
+* `Home Assistant <https://www.home-assistant.io/>`_ with the BTHome integration running
```

Comment:
Strictly speaking it is not limited to using Home Assistant. Any BTHome compatible listener will work. Perhaps just list Home Assistant as an example?